README.rst

```diff
@@ -1,7 +1,7 @@
 INF200 Exersices:
 =================
 
-FIRST_NAME SURNAME:
+Jørgen Kongsro:
```

Comment:
Hi Daniel, there seem to be something strange going on with my git activity. My git learning curve is a bit steep at the moment, and the wrong file tidy_code.py was submitted for review. I have attached and moved the files to /src/YOUR_NAME_ex/ex_01/. I hope this is sufficient, otherwise I need some help at the tutorials on Friday.